src/zampy/datasets/era5.py

```diff
@@ -1,247 +1,60 @@
         Variable(name="u10", unit=unit_registry.meter_per_second),
         Variable(name="v10", unit=unit_registry.meter_per_second),
-    )
+    ]
 
     # variable names used in cdsapi downloading request
-    variable_names = (
-        "mean_total_precipitation_rate",
-        "surface_thermal_radiation_downwards",
-        "surface_solar_radiation_downwards",
-        "surface_pressure",
-        "10m_u_component_of_wind",
-        "10m_v_component_of_wind",
-    )
-
-    license = "cc-by-4.0"
-    bib = """
-    @article{hersbach2020era5,
-        title={The ERA5 global reanalysis},
-        author={Hersbach, Hans et al.},
-        journal={Quarterly Journal of the Royal Meteorological Society},
-        volume={146},
-        number={730},
-        pages={1999--2049},
-        year={2020},
-        publisher={Wiley Online Library}
-        }
-    """
-
-    def download(
-        self,
-        download_dir: Path,
-        time_bounds: TimeBounds,
-        spatial_bounds: SpatialBounds,
-        variable_names: List[str],
-        overwrite: bool = False,
-    ) -> bool:
-        validation.validate_download_request(
-            self,
-            download_dir,
-            time_bounds,
-            spatial_bounds,
-            variable_names,
-        )
-
-        download_folder = download_dir / self.name
-        download_folder.mkdir(parents=True, exist_ok=True)
-
-        utils.cds_request(
-            dataset="reanalysis-era5-single-levels",
-            variables=variable_names,
-            time_bounds=time_bounds,
-            spatial_bounds=spatial_bounds,
-            path=download_folder,
-            overwrite=overwrite,
-        )
-
-        write_properties_file(
-            download_folder, spatial_bounds, time_bounds, variable_names
-        )
-
-        return True
-
-    def ingest(
-        self,
-        download_dir: Path,
-        ingest_dir: Path,
-        overwrite: bool = False,
-    ) -> bool:
-        download_folder = download_dir / self.name
-        ingest_folder = ingest_dir / self.name
-        ingest_folder.mkdir(parents=True, exist_ok=True)
-
-        data_file_pattern = "era5_*.nc"
-        data_files = list(download_folder.glob(data_file_pattern))
-
-        for file in data_files:
-            convert_to_zampy(
-                ingest_folder,
-                file=file,
-                overwrite=overwrite,
-            )
-
-        copy_properties_file(download_folder, ingest_folder)
-
-        return True
-
-    def load(
-        self,
-        ingest_dir: Path,
-        time_bounds: TimeBounds,
-        spatial_bounds: SpatialBounds,
-        resolution: float,
-        regrid_method: str,
-        variable_names: List[str],
-    ) -> xr.Dataset:
-        files: List[Path] = []
-        for var in self.variable_names:
-            if var in variable_names:
-                files += (ingest_dir / self.name).glob(f"era5_{var}*.nc")
+    cds_var_names = {
```

Comment:
I think this is the wrong way around. We'd want to download a variable from a dataset using the _standard names_. This is also reflected by the docstrings in ECMWFDataset. Modifications have to be made in ECMWFDataset as well.

The following should work (approximately):
```py
era5 = Era5()
era5.download(
    variables=["eastward_component_of_wind"],  # NOT "10m_u_component_of_wind"
)
```

This way we can also do for example:
```py
cerra = Cerra()
cerra.download(
    variables=["eastward_component_of_wind"],
)
```

Reply:
Thanks for the comment! Indeed we should use the zampy variables names. I've updated the download functions and also the load functions, to make sure that they all work with zampy names.